docs/contributing-to-analyses/determining-requirements/determining-software-requirements.md

```diff
@@ -93,15 +93,16 @@ library(svglite)
 #### Restoring dependencies
 
 If you are switching between your computer and Amazon Web Services or running another contributor's module, it can be helpful to restore dependencies using a module's lockfile.
-To do so, use the following command ([reference](https://rstudio.github.io/renv/reference/restore.html)):
+To do so, use [`renv::restore()` function](https://rstudio.github.io/renv/reference/restore.html):
 
 ```r
 renv::restore()
 ```
 
-## Managing software dependencies in Python with conda
+## Managing software dependencies in Python and other languages with conda
```

Comment:
I'm on the fence for "other languages" since it might imply R? I might prefer "other software dependencies" but that's too long I think..

Reply:
I changed this to just "with conda" and left the better explanation for the text